src/client/Main.ts

```diff
@@ -22,13 +22,24 @@ import { HelpModal } from "./HelpModal";
 import { GameType } from "../core/game/Game";
 
 class Client {
+  private spModal: SinglePlayerModal;
+  private hostModal: HostPrivateLobbyModal;
+  private joinModal: JoinPrivateLobbyModal;
+  private helpModal: HelpModal;
+  private flagInput: FlagInput;
+
+  closeAllModals() {
+    this.spModal.close();
+    this.hostModal.close();
+    this.helpModal.close();
+    this.joinModal.close();
+    this.flagInput.close();
+  }
```

Comment:
```suggestion
  private closeAllModals() {
    this.spModal.close();
    this.hostModal.close();
    this.helpModal.close();
    this.joinModal.close();
    this.flagInput.close();
  }
```

additionally further suggest deleting these lines from this location and re-inserting just prior to 
```
  private async handleJoinLobby(event: CustomEvent) {
```
which is currently line 143. Placement of a function in the middle of the property definitions is not ideal. Should be placed with the other functions, after initialize().

Reply:
@NewHappyRabbit, I know this is an old PR, but it's still relevant. I'd like to clear out as many of these old PRs as possible, so if you are able to make the requested changes soon, that would be helpful. Thank you!